### DISCUSSION TITLE
Welcome to Chat_Room Discussions!

### DISCUSSION THREAD

**Discussion by Sabirjankp** [Diamond-Family/Chat_Room#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sabirjankp** [comment]
Dear Members please give some bullet point about how can we enhance this app more than this simple demo.

**Comment by HamdullahHamdard** [comment]
ما ویله چی دا peer to peer څنکه کار دی

مجبوره یو سرور ته هم یو لیست جوړو کړو

**Comment by hazratulla** [comment]
السلام علیکم ورحمه الله 
دغه Ip ادرسونه څه رول لری زموږ په پروژه کی

**Comment by hazratulla** [comment]
السلام علیکم ورحمه الله 
دغه Ip ادرسونه څه رول لری زموږ په پروژه کی

**Reply by HamdullahHamdard** [reply]
وبخښه وروره موږ Ip ادرس خپل په پروژه کی ځکه نه دی کارولی چی موږ لوکل پروژه جوړه کړی.

**Comment by HamdullahHamdard** [comment]
موږ خپل په پروژه کی فورټ ادرسونه کارولی چی هغه د کلاینټونو د وصلیدو کار ترسره کوی.

**Comment by AsadShahi** [comment]
teacher said pull request must be minimum twice?

**Reply by HamdullahHamdard** [reply]
Yas create it.

**Comment by AsadShahi** [comment]
بخاطر ساخت چت روم در لوکل سیستم از
multi threading and java  socket from library
استفاده شده در این پروژه.
